src/engine/sys/sdl_glimp.cpp

```diff
@@ -785,20 +785,28 @@ static bool GLimp_CreateContext( const glConfiguration &configuration )
 
 /* GLimp_DestroyWindowIfExists checks if window exists before
 destroying it so we can call GLimp_RecreateWindowWhenChange even
-if no window is created yet.
-
-It is assumed width, height and other things like that are unchanged,
-given vid_restart is called when changing those and then destroying
-the window before calling this. */
+if no window is created yet. */
 static bool GLimp_RecreateWindowWhenChange( const bool fullscreen, const bool bordered, const glConfiguration &configuration )
 {
+	/* Those Values doen't contribute to anything when the window
+	is't created yet, but we need them to be set to silence warnings
+	about comparing undefined variables. */	
```

Comment:
static variables are always initialized, so the comment about warnings is not true.

LGTM

Reply:
Right, I edited the comment.